packages/contractkit/src/utils/web3-utils.ts

```diff
@@ -32,6 +34,14 @@ export const estimateGas = async (
     if (called.startsWith('0x08c379a')) {
```

Comment:
Unrelated to your PR but would be great if this was extracted to a named variable